sites/includes/metadata.html

```diff
@@ -1 +1,9 @@
-<meta name="description" content="{% if page.description[page.lang] %}{{ page.description[page.lang] }}{% else %}{{ site.site-description[page.lang] }} {% endif %}" />
\ No newline at end of file
+<meta name="description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description[page.lang] }} {% endif %}" />
+<meta name="author" content="{% if page.creator %}{{ page.creator }}{% else %}{{ site.creator[page.lang] }} {% endif %}"/>
+<meta name="dcterms.title" content="{{ page.title }}"/>
+<meta name="dcterms.description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description[page.lang] }} {% endif %}"/>
+<meta name="dcterms.creator" content="{% if page.creator %}{{ page.creator }}{% else %}{{ site.creator[page.lang] }} {% endif %}"/>
+<meta name="dcterms.language" title="ISO639-2/T" content="{{ i18nText-language | downcase | truncate: 3, '' }}"/>
+<meta name="dcterms.subject" title="gccore" content="{{ page.subject }}"/>
+<meta name="dcterms.issued" title="W3CDTF" content="{{ page.dateIssued }}"/>
+<meta name="dcterms.modified" title="W3CDTF" content="{% if page.dateModified %}{{ page.dateModified }}{% else %}{{ page.dateIssued }}{% endif %}"/>
```

Comment:
Check if we can leverage the jekyll page variable: `page.date`

ref. https://jekyllrb.com/docs/variables/

Reply:
> Check if we can leverage the Jekyll page variable: `page.date`

After a few test the `page.date` has to be explicitly declare on a page for Jekyll to use this value

> To Check: I do think we named the front-matter modified and issued.

Right now it seems that `dateIssued` and `dateModified` are used in the splash page. Should make a separate PR to rename these variables.

I think for now this should be ready to merge.

Reply:
That is fine. Thanks @delisma for checking how those page properties was working.